packages/piro/test/Piro_TempusSolver_UnitTests.cpp

```diff
@@ -361,4 +362,58 @@ TEUCHOS_UNIT_TEST(Piro_TempusSolver, ObserveFinalSolution)
   TEST_FLOATING_EQUALITY(observer->lastStamp(), finalTime, tol);
 }
 
+TEUCHOS_UNIT_TEST(Piro_TempusSolver, ObserveFinalSolutionNoWrapper)
+{
+  const RCP<Thyra::ModelEvaluatorDefaultBase<double> > model = defaultModelNew();
+  auto observer = Teuchos::rcp(new MockTempusObserver<double>());
+
+  const double initialTime = 0.0;
+  const double finalTime = 0.1;
+  const double timeStepSize = 0.05;
+
+  const RCP<ParameterList> appParams(new ParameterList("params"));
+  auto& tempusPL = appParams->sublist("Tempus");
+  tempusPL.set("Integrator Name", "Demo Integrator");
+  tempusPL.sublist("Demo Integrator").set("Integrator Type", "Integrator Basic");
+  tempusPL.sublist("Demo Integrator").set("Stepper Name", "Demo Stepper");
+  tempusPL.sublist("Demo Integrator").sublist("Solution History").set("Storage Type", "Unlimited");
+  tempusPL.sublist("Demo Integrator").sublist("Solution History").set("Storage Limit", 20);
+  tempusPL.sublist("Demo Integrator").sublist("Time Step Control").set("Initial Time", initialTime);
+  tempusPL.sublist("Demo Integrator").sublist("Time Step Control").set("Final Time", finalTime);
+  tempusPL.sublist("Demo Integrator").sublist("Time Step Control").set("Minimum Time Step", fixedTimeStep);
+  tempusPL.sublist("Demo Integrator").sublist("Time Step Control").set("Initial Time Step", fixedTimeStep);
+  tempusPL.sublist("Demo Integrator").sublist("Time Step Control").set("Maximum Time Step", fixedTimeStep);
+  tempusPL.sublist("Demo Stepper").set("Stepper Type", "Backward Euler");
+  tempusPL.sublist("Demo Stepper").set("Zero Initial Guess", false);
+  tempusPL.sublist("Demo Stepper").set("Solver Name", "Demo Solver");
+  tempusPL.sublist("Demo Stepper").sublist("Demo Solver").sublist("NOX").sublist("Direction").set("Method","Newton");
+
+  auto solver = tempusSolver(appParams,model,Teuchos::null,observer);
+
+  // Check that the piro integrator did NOT wrap our observer in a tempus adapter
+  auto piro_integrator = solver->getPiroTempusIntegrator();
+  auto tempus_integrator = piro_integrator->getIntegrator();
+  auto basic_integrator = Teuchos::rcp_dynamic_cast<const Tempus::IntegratorBasic<double>>(tempus_integrator);
```

Comment:
@ccober6 I noticed that `Tempus::Integrator` does not offer `getObserver`, but the derived classes (basic, fwdSesn, and adjSens) all do (and it's virtual). Was the intent to add it as an interface to the base class too, or was there a reason not to add it to the base class interface?

Here it would save me a cast, but it's not a big deal.